.github/workflows/publish-release.yml

```diff
@@ -27,20 +20,12 @@ jobs:
 
       - name: Install Android SDK
         run: ./.github/android-sdk.sh
-        if: matrix.os == 'macos-latest'
+
+      - name: Build release
+        run: ./gradlew assemble
```

Comment:
Adding this as an extra step. I've found that building and publishing in one step can lead to having to staging repos on sonatype if the build is slow (it builds one artifacts, publishes it, builds another, a lot of time has passed, publishes the second artfact to a new repo). This then leads to `closeAndReleaseRepository` to fail. This way the following `publish` can just use the already built binaries